app/Jobs/Bank/MatchBankTransactions.php

```diff
@@ -81,6 +81,7 @@ public function __construct(int $company_id, string $db, array $input)
      * Execute the job.
      *
      */
+    // TODO: what are these categories, and for what do we need them
```

Comment:
These are Yodlee Default Bank Categories, we check here is there is a cached set for reuse.

Now that we have a new provider, we'll need to use additional logic to see whether the appropriate set of categories are present.
 For Nordigen you'll want to do a

Reply:
if i read correctly, nordigen bank account api does not have categories., they have additional apis for this i guess to remember

Reply:
ok, lets just ensure we set null for the category prop.

Reply:
yes, we do
![image](https://github.com/invoiceninja/invoiceninja/assets/52678724/b99780d3-d5d4-401e-ba7f-3def0bd9b347)

Reply:
you may look at the category_type prop, if we should fill it or not?!

Reply:
field-info: Might be used by the financial institution to transport additional transaction related information

Reply:
I don't think we have anywhere to store this information currently. Perhaps we can look into this in a second pass if needed?

Reply:
sounds great. I close this threat.